docs/1.16/prisma-client/api-reference/realtime-GO-rsc10.mdx

```diff
@@ -0,0 +1,11 @@
+import Collapse from 'components/Markdown/Collapse'
+
+export const meta = {
+  title: 'Realtime (Go)',
+  position: 50,
+  technology: "go",
+  technologyOrder: 2,
+  articleGroup: "Realtime",
+}
+
+Coming Soon
```

Comment:
do we have a feature request we can link here? if no, can you maybe create a simple one?